src/DtoGenerator/DtoGenerator.Logic/Infrastructure/EntityParser.cs

```diff
@@ -154,13 +160,31 @@ private static EntityMetadata FromSyntaxTree(SyntaxTree syntaxTree)
                     SyntaxNode = p,
                     IsCollection = IsCollection(p),
                     IsRelation = IsRelation(p),
-                    RelatedEntityName = IsRelation(p) ? GetRelatedEntity(p) : null
+                    RelatedEntityName = IsRelation(p) ? GetRelatedEntity(p) : null,
+                    AttributesList = p.AttributeLists
```

Comment:
Would you mind filtering only attributes from System.Data.Annotations here? I think it might be a better option than getting all the attributes? If someone uses EF database first, I think it generates additional attributes which are more-less useless in DTOs?